recipes/pyppeteer/setup.patch

```diff
@@ -0,0 +1,34 @@
+From 08897b66cf89c405ba8d9edbe18580e5126ef844 Mon Sep 17 00:00:00 2001
```

Comment:
Why this patch is needed?

Reply:
to get it to build, right @kiwi0fruit ?

Reply:
Not really. It was more a preventive measure so I would predict how it behaves and make sure it won't give bugs.

Reply:
Ahhhh ok. I'd say we remove it. I don't think it is doing that much to make things safer and probably creates more maintenance.